src/right-field/RightField.jsx

```diff
@@ -267,6 +272,27 @@ async function executeQuery(query, eventEmitter, resultAdder) {
   }
 }
 
+/**
+ * Tries to fetch a query onauthenticated, if it fails it tries to fetch it authenticated.
+ * @param {String || URL} argument a URL or a string that should be fetched
+ * @returns {Response} the response from the fetch
+ */
+async function authenticationFetch(argument) {
```

Comment:
This feels very inefficient. There's no other way of fixing this?
Also, when commenting on stuff, also include the reasoning on why you do it like this, in this case because there's an issue with authenticated fetches on open data